src/panels/config/cloud/alexa/cloud-alexa.ts

```diff
@@ -117,33 +125,80 @@ class CloudAlexa extends LitElement {
       target.push(html`
         <ha-card>
           <div class="card-content">
-            <state-info
-              .hass=${this.hass}
-              .stateObj=${stateObj}
-              secondary-line
-              @click=${this._showMoreInfo}
-            >
-              ${entity.interfaces
-                .filter((ifc) => !IGNORE_INTERFACES.includes(ifc))
-                .map((ifc) =>
-                  ifc.replace("Alexa.", "").replace("Controller", "")
-                )
-                .join(", ")}
-            </state-info>
-            <ha-formfield
-              .label=${this.hass!.localize(
-                "ui.panel.config.cloud.alexa.expose"
-              )}
-              .dir=${computeRTLDirection(this.hass!)}
-            >
-              <ha-switch
-                .entityId=${entity.entity_id}
-                .disabled=${!emptyFilter}
-                .checked=${isExposed}
-                @change=${this._exposeChanged}
+            <div class="top-line">
+              <state-info
+                .hass=${this.hass}
+                .stateObj=${stateObj}
+                secondary-line
+                @click=${this._showMoreInfo}
+              >
+                ${entity.interfaces
+                  .filter((ifc) => !IGNORE_INTERFACES.includes(ifc))
+                  .map((ifc) => ifc.replace(/(Alexa.|Controller)/g, ""))
+                  .join(", ")}
+              </state-info>
+              <ha-button-menu
+                corner="BOTTOM_START"
+                .entityId=${stateObj.entity_id}
+                @action=${this._exposeChanged}
               >
-              </ha-switch>
-            </ha-formfield>
+                <mwc-icon-button
+                  slot="trigger"
+                  class=${classMap({
+                    exposed: isExposed!,
+                    "not-exposed": !isExposed,
+                  })}
+                  .title=${this.hass!.localize(
+                    "ui.panel.config.cloud.alexa.expose"
```

Comment:
I know I said to call it this, but now we have identical keys for google/alexa, should we combine them?

Reply:
I want to combine the two pages in the future, as they are almost identical